tests/mock-snapd.c

```diff
@@ -796,6 +796,22 @@ mock_snapd_add_change (MockSnapd *self)
     return add_change (self);
 }
 
+void
+mock_change_set_kind (MockChange *change, const gchar *kind)
+{
+    g_free (change->kind);
+    change->kind = g_strdup (kind);
+}
+
+void
+mock_change_add_data (MockChange *change, const JsonNode *data)
```

Comment:
Drop the const so we don't need the cast later.

Reply:
Done.